packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1535,6 +1535,23 @@ image = sana(
 ) `,
 ];
 
+export const videoprism = (model: ModelData): string[] => [
```

Comment:
If each model is in its own repo, then you can use `model.id` in the snippet to get the model name and retrieve the weights.